app/main.py

```diff
@@ -91,57 +87,3 @@ def simple_route() -> dict[str, str]:
     """
     logger.info('Hello World')
     return {'Hello': 'World'}
-
-
-@app.post('/db/test', status_code=status.HTTP_201_CREATED)
-async def test_db_create(
```

Comment:
Curious why you removed these - Did you get the DB working with unit tests or with the other endpoints?

Reply:
I don't follow this either.  Does it have something to do with these changes making these routes require jwt auth?